play.py

```diff
@@ -57,7 +57,8 @@ def get_action(self, obs):
 
 
 if __name__ == '__main__':
-    env = gym.make('Mario-Kart-Royal-Raceway-v0')
+    env = gym.make('Mario-Kart-Luigi-Raceway-v0')
+    #env = gym.make('Mario-Kart-Royal-Raceway-v0')
```

Comment:
can you remove this extra comment line please

Reply:
ok